app/controllers/Switches.scala

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import actions.EmergencySwitchChangeAccess
+import config.{Off, On, SwitchState}
+import controllers.Login._
+import play.Logger
+
+object Switches {
+
+  private def errorMessage(state: SwitchState)  = s"Failed to update Emergency switch to ${state.name}. Contact digitalcms.dev@theguardian.com for more help."
+  private def success(state: SwitchState)  = s"Emergency switch updated to ${state.name}"
+
+  def index = AuthAction { req =>
+    Ok(views.html.switches.switchValues(config.Switches.allSwitches))
+  }
+
+  def emergencyOn = EmergencySwitchChangeAccess { req =>
+    try {
+      config.Switches.setEmergencySwitch(On)
+      Ok(views.html.switches.switchChange(success(On)))
+    } catch {
+      case e: Exception =>
+        Logger.error(e.getMessage)
```

Comment:
Good to log the `e` as the second argument when you have one available.

That said, these are precisely duplicates of the logging done in `setEmergencySwitch` so I'm not sure what value these add at the moment. As I mentioned on that fn, maybe better to have it return a value rather than re-throwing the error and forcing you catch it here as well?